cmd/collector/app/server/zipkin_test.go

```diff
@@ -57,3 +63,197 @@ func TestSpanCollectorZipkin(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, response)
 }
+
+func TestSpanCollectorZipkinTLS(t *testing.T) {
+	testCases := []struct {
+		name                  string
+		TLS                   tlscfg.Options
```

Comment:
`serverTLS` seems like a better name to me, esp. given there's a `clientTLS` test case attribute; what do you think?

Reply:
Ok